$NetBSD: patch-af,v 1.2 2011/07/02 21:20:47 markd Exp $

build with gcc4.3
curl/types.h is not used (empty) since curl-7.12.0 and removed at curl-7.21.7.

--- examples/protocol.cpp.orig	2006-05-10 18:05:42.000000000 +0000
+++ examples/protocol.cpp
@@ -9,10 +9,10 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string>
+#include <cstring>
 #include <map>
 #include <expat.h>
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 
 using namespace std;
